### PR TITLE
EN-2184 

### PR DESCRIPTION
**Pull request by rzsouza**
Added check to avoid deletion for admin user.

Code is working now but might need improvements in the error handling for FE

### CODE REVIEW

**Comment by rzsouza** [comment]
Last commit added a specific validation exception that can be used by FE to display specific error

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/web/user/UserController.java [hunk 1]
```diff
@@ -163,6 +165,15 @@ public void setUserValidator(UserValidator userValidator) {
         return new ResponseEntity<>(new RestResponse(result), HttpStatus.OK);
     }
 
+    private BindingResult createBindingResult(@PathVariable String username) {
+        Map<String, String> map = new HashMap<>();
+        map.put("username", username);
+        BindingResult bindingResult = new MapBindingResult(map, "username");
+        ObjectError error = new ObjectError("username", "Sorry. You can't delete the administrator user");
```
Comment (eugeniosant) [inline engine/src/main/java/org/entando/entando/web/user/UserController.java]: you have to use message.properties file for text messages

Comment (rzsouza) [reply engine/src/main/java/org/entando/entando/web/user/UserController.java]: Fixed

**Comment by entando-service** [comment]
All is well

**Pull request merged**